ide/dlight.nativeexecution/external/binaries-list

```diff
@@ -15,3 +15,4 @@
 # specific language governing permissions and limitations
 # under the License.
 4B4DCA62F8C4A1954AE6D286955C36CC50B8CC3A exechlp-1.2.zip
+C6E6C873F18A4E9FB91F18C5342580711B0339E7 https://dist.apache.org/repos/dist/dev/netbeans/native/netbeans-nativeexecution/1-r24aefa9/nativeexecution-external-binaries-1-24aefa9.zip nativeexecution-external-binaries-1-24aefa9.zip
```

Comment:
Release is done and binaries propagated to archive, so this can now be updated to:

```suggestion
C6E6C873F18A4E9FB91F18C5342580711B0339E7 https://archive.apache.org/dist/netbeans/native/netbeans-nativeexecution/1-24aefa9/nativeexecution-external-binaries-1-24aefa9.zip nativeexecution-external-binaries-1-24aefa9.zip
```